Modules/Office365DSC/DSCResources/MSFT_SPOSite/MSFT_SPOSite.psm1

```diff
@@ -875,6 +875,22 @@ function Set-SPOSiteConfiguration
             }
 
             $CurrentParameters = $PSBoundParameters
+            if($CurrentParameters.Template -eq "GROUP#0")
+            {
+                if ($CurrentParameters.ContainsKey("Title")) { $null = $CurrentParameters.Remove("Title") }
```

Comment:
Why are you doing $null = all over the place? we should simply remove the parameter.

Reply:
as discussed :)